db/seeds.rb

```diff
@@ -9,9 +9,9 @@
 #   end
 quiz = Quiz.create!(
   topic: "Ruby on Rails Basics",
-  difficulty: "Intermmediate",
+  difficulty: :easy,
   study_duration: 30,
-  detail_level: "High",
+  detail_level: "high",
   number_of_questions: 2,
   score: 0
 )
```

Comment:
I think these changes are correct because it follows `enum` defined in `app/models/quiz.rb`. Also, using string and symbol in this case are interchangeable.